frontend/src/components/widget/index.js

```diff
@@ -28,11 +28,21 @@ const Widget = observer(({
   widgetName,
   styles,
 }) => {
-  const { settingsStore: { toggleWidget, widgets }} = useStore()
+  const {
+    videoPlayerStore: { onShowModal, isStreamStarted },
+    settingsStore: { toggleWidget, widgets }
+  } = useStore()
 
   const handleWidgetClose = (e) => {
     e.preventDefault()
-    toggleWidget(widgetName)
+    const toggleWidgetCallback = () => toggleWidget(widgetName)
+
+    if(widgetName === 'screen' && isStreamStarted) {
```

Comment:
Do we have a fixed list of screens stored somewhere?

Reply:
sorry, probably I didn't get your question properly
we don't have fixed list of screens
in this case is used `screen` widget name. widgets to show on dashboard can be configured in settings tab